eskipfile/remote.go

```diff
@@ -23,17 +23,17 @@ type remoteEskipFile struct {
 }
 
 type RemoteWatchOptions struct {
-	remoteFile    string
-	threshold     int
-	verbose       bool
-	failOnStartup bool
+	RemoteFile    string
+	Threshold     int
+	Verbose       bool
+	FailOnStartup bool
```

Comment:
@xavivars Thank you for the improvement. Making this fields public makes them part of the module interface (and its not very useful when private) and therefore it would be nice to have a onliner doc comment for each field like e.g. https://github.com/zalando/skipper/blob/9c3be7ba13cb3520e07cd419dc971b2521951bda/proxy/proxy.go#L142-L148

Reply:
Resolved!